test/conformance/tests/sriov_operator.go

```diff
@@ -69,7 +69,16 @@ var _ = Describe("[sriov] operator", func() {
 		isSingleNode, err := cluster.IsSingleNode(clients)
 		Expect(err).ToNot(HaveOccurred())
 		if isSingleNode {
+			if discovery.Enabled() {
+				disableDrain, err := getNodeDrain()
+				Expect(err).ToNot(HaveOccurred())
+				if !disableDrain {
+					Skip("SriovOperatorConfig DisableDrain property must be enabled in a single node environment")
+				}
+			}
 			snoTimeoutMultiplier = 1
+			err = setNodeDrain()
```

Comment:
we need to rollback this after we finish running the tests.

Reply:
Done